pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: crypto
-version: 2.1.1
+version: 2.1.2
```

Comment:
This is a perfect case where it makes sense to just increment the build number instead of the patch. So: `2.1.1+1`.

Reply:
roger!